pages/docs/integrations/grouparoo-intercom.mdx

```diff
@@ -11,132 +11,218 @@ Grouparoo's [Intercom](https://www.intercom.com/) Plugin enables you to export c
+  <tr>
+    <td>Option</td>
+    <td>Description</td>
+  </tr>
+  <tr>
+    <td>
+      <strong>Access Token</strong> (required)
+    </td>
+    <td>Access token from your private app in the developer hub.</td>
+  </tr>
+</table>
+
+For more information on options for keeping your credentials secure, visit our documentation on [secrets](https://www.grouparoo.com/docs/support/secrets).
 
-The Intercom Destination supports all three sync modes:
+## Intercom Export Contacts
 
-- **Sync** (`sync`): Add, update, and remove Records as needed.
-- **Additive** (`additive`): Add and update Records as needed, but do not remove anybody.
-- **Enrich** (`enrich`): Only update Records that already exist in the Destination. Do not add or remove anybody.
+This Destination enables you to sync your Grouparoo Records to Contacts in Intercom as either Users, Leads, or a dynamic combination of both depending on data.
 
-You can set the desired Sync Mode through the `syncMode` Property in the config file:
+Companies use Grouparoo to keep their Intercom Contacts up-to-date in order to communicate with their customers through a wide array of channels using one tool.
 
-```js
-syncMode: "additive",
-```
+To [create a Destination](https://www.grouparoo.com/docs/config/destinations), you make choices about what data to sync.
+
+### Sync Modes
+
+A Destination’s sync mode determines what to update in a Destination. The following are supported by Intercom:
+
+- **Sync**: Add, update, and remove Intercom Contacts as needed.This option looks for new records, changes to existing records, and deletions.
+- **Additive**: Add and update Intercom Contacts as needed, but do not remove anybody. This option looks for new Contacts to add to Intercom and changes to existing Intercom Contacts, but does not keep track of deletions.
+- **Enrich**: Only update those Contacts that already exist in Intercom. Do not add or remove Contacts. This option will only update existing Contacts in Intercom.
 
 ### Destination Options
 
-Here are the available options for an Intercom Destination:
-
-<OptionsList
-  options={{
-    creationMode: {
-      options: ["User", "Lead", "Lifecycle"],
-      description:
-        "How should Grouparoo create Intercom contacts? Note that this only applies if using a `syncMode` that can add new contacts.",
-    },
-    removalMode: {
-      displayName: "Removal",
-      options: ["Archive", "Delete"],
-      description:
-        "How should Grouparoo remove Intercom contacts? Note that this only applies if using a `syncMode` that can remove existing contacts.",
-    },
-  }}
+To further clarify how Intercom should sync data, select how data should be synced.
+
+<ImageInBrowserFrame
+  src="/images/docs/integrations/intercom/intercom-destination-options.png"
+  alt="Intercom Destination Options"
+  width={1625}
+  height={470}
 />
 
-You can configure these options by setting them inside the `options` object of the config file:
+<table className="docTable">
```

Comment:
This docTable class should be moved from markdown here (and in the 97-things blog post) into the scss file to have the class available thought the site.

Reply:
I moved the docTable and the notesTable (I needed some differences for the docTable class, so I didn't use the exact same styling) and checked they still work.